### ISSUE TITLE
Next.js Support

### ISSUE THREAD

**Issue by owebudde** [taichi-t/weather-react-icons#4]
**Got the following error:**
```
./node_modules/weather-react-icons/lib/css/weather-icons.css
Global CSS cannot be imported from within node_modules.
Read more: https://err.sh/next.js/css-npm
Location: node_modules/weather-react-icons/lib/WeatherIcon.js
```

**Solution:**
Compile the package and set the npmignore to send only the lib directory.
More info [here](https://github.com/vercel/next.js/blob/master/errors/css-npm.md)

**Comment by taichi-t** [comment]
Thank you for posting the issue. I've fixed #4, more detail please see #5.

next.js user can use this package as follows
```javaScript
//_app.js
import "weather-react-icons/lib/css/weather-icons.css"
```
```javaScript
//component.js
import {WeatherIcon} from "weather-react-icons"
```
I've tested with next.js 10.0.6 and node.js v12.13.1, it works fine. The repository is [here](https://github.com/taichi-t/demo-for-weahter-react-icons-4.0.1-with-nextjs)
If there are any questions, feel free to leave comments.

**Comment by owebudde** [comment]
Awesome! Thank you!
